tests/promptlab/test_promptlab_model.py

```diff
@@ -0,0 +1,43 @@
+from unittest import mock
+
+import pandas as pd
+import pytest
+
+from mlflow._promptlab import _PromptlabModel
+from mlflow.entities.param import Param
+
+
+@pytest.fixture
+def data():
+    return pd.DataFrame(
+        data=[
+            {"thing": "books"},
+            {"thing": "coffee"},
+            {"thing": "nothing"},
+        ]
+    )
+
+
+def test_promptlab_prompt_replacement(data):
```

Comment:
```suggestion
def test_promptlab_prompt_replacement():
    data = pd.DataFrame(
        data=[
            {"thing": "books"},
            {"thing": "coffee"},
            {"thing": "nothing"},
        ]
    )
```

I'd directly use `pd.DataFrame`. We can add fixture later if necessary.